templates/flows/flow_create.haml

```diff
@@ -61,7 +61,7 @@
       setTimeout(function() {
         removeFakeCreateButton();
         fakeCreateButton();
-      }, 50)
+      }, 500)
```

Comment:
Tem algum motivo específico para o timeout ser de 500ms? Se for o caso, acho interessante declarar uma constante com esse valor pra explicitar o significado/motivo desse número ou um comentário informando o que ele significa, pra evitar "número mágicos"

Reply:
O tempo de timeout é para dar tempo para a página carregar, de forma que os elementos que são buscados com querySelector estejam na DOM.